### DISCUSSION TITLE
Social login: "No resumable session could be found in the HTTP Header."

### DISCUSSION THREAD

**Discussion by saulofilho** [ory/kratos#2108] [Q&A] [answered]
Hi, guys.

I'm trying to implement a Social Login with Google but I'm receiving this error:
`"code": 400,"status":"Bad Request","reason":"No resumable session could be found in the HTTP Header.","debug":"key ory_kratos_oidc_auth_code_session does not exist in cookie: map[]..."`

What `ory_kratos_oidc_auth_code_session` stands for?

Thank you.

**Comment by aeneasr** [comment, marked as answer]
Hello, this can sometimes happen when something goes wrong with cookies or the browser has a weird plugin installed. Just try the flow again, it should then work! If not, delete the cookies for the domain and try again

**Reply by saulofilho** [reply]
Nice, thanks. I really tried this before. But I gonna try it again. If has any update, I'll post here.

**Comment by saulofilho** [comment]
Hi. It's me again.

I'm still stuck in this error.

Here it goes what I'm doing:

**1.**
Inits the browser's flow to get the flow id:
GET `http://local.angeplus.com.br/iam-svc/public/self-service/registration/browser`

Cookie was setted:
Name: `csrf_token_6e30977bbcc76f0a13b9753cab1b4760c9b51745d668cc47faf7ef7beab35bce`
Value:  `Vydg+77NYElkBa9GrHiJNrjLq3L0I3zrms35IdS87bI=`

**2.**
Than, I'm pass the flow:
GET `http://local.angeplus.com.br/iam-svc/public/self-service/registration/flows?id=86c5581b-7e23-446b-afa6-0e6821d23fac`

Responds with `200` where I can get the csrf_token value:
```
{
	"id": "86c5581b-7e23-446b-afa6-0e6821d23fac",
	"type": "browser",
	"expires_at": "2022-01-06T20:15:17.58475Z",
	"issued_at": "2022-01-06T20:05:17.58475Z",
	"request_url": "http://local.angeplus.com.br/self-service/registration/browser",
	"ui": {
		"action": "http://local.angeplus.com.br/iam-svc/public/self-service/registration?flow=86c5581b-7e23-446b-afa6-0e6821d23fac",
		"method": "POST",
		"nodes": [
			{
				"type": "input",
				"group": "default",
				"attributes": {
					"name": "csrf_token",
					"type": "hidden",
					"value": "5eikUYcFsVkV+gYEXfOEYs13z/nUi3h7LHNAAGtge6eyz8SqOcjREHH/qULxiw1UdbxkiyCoBJC2vrkhv9yWFQ==",
```

**3.**
After that, I'm trying to complete the flow:
POST `http://local.angeplus.com.br/iam-svc/public/self-service/registration?flow=86c5581b-7e23-446b-afa6-0e6821d23fac`

Form URL Encoded:
`method=oidc&provider=google&csrf_token=5eikUYcFsVkV%2BgYEXfOEYs13z%2FnUi3h7LHNAAGtge6eyz8SqOcjREHH%2FqULxiw1UdbxkiyCoBJC2vrkhv9yWFQ%3D%3D`

Cookie was setted:
Name: `ory_kratos_continuity`
Value: `MTY0MTQ5OTc0OHxEdi1CQkFFQ180SUFBUkFCRUFBQVhfLUNBQUVHYzNSeWFXNW5EQ01BSVc5eWVWOXJjbUYwYjNOZmIybGtZMTloZFhSb1gyTnZaR1ZmYzJWemMybHZiZ1p6ZEhKcGJtY01KZ0FrTTJFek5UZ3laRGN0Wm1JNE1TMDBNamd4TFRnd1pHTXRNamd3T1Rrell6VXdaRFpofMvmcjUyEASYMKdVYK2DYAurKzQUqKDVEDowFFEfKFcy`

Receives:
Location: 
```
https://accounts.google.com/o/oauth2/v2/auth?claims=%7B%22id_token%22%3A%7B%22email%22%3A%7B%22essential%22%3Atrue%7D%2C%22email_verified%22%3A%7B%22essential%22%3Atrue%7D%7D%7D&client_id=410142226801-hfn6eon0ucgk1kiddc36ljrmuhsngsr6.apps.googleusercontent.com&redirect_uri=http%3A%2F%2Flocal.angeplus.com.br%2Fiam-svc%2Fpublic%2Fself-service%2Fmethods%2Foidc%2Fcallback%2Fgoogle&response_type=code&scope=email+profile+openid&state=c8acef4d-989b-4cf4-a259-5630f57fb279
```
Set-Cookie: 
```
ory_kratos_continuity=MTY0MTQ5OTc0OHxEdi1CQkFFQ180SUFBUkFCRUFBQVhfLUNBQUVHYzNSeWFXNW5EQ01BSVc5eWVWOXJjbUYwYjNOZmIybGtZMTloZFhSb1gyTnZaR1ZmYzJWemMybHZiZ1p6ZEhKcGJtY01KZ0FrTTJFek5UZ3laRGN0Wm1JNE1TMDBNamd4TFRnd1pHTXRNamd3T1Rrell6VXdaRFpofMvmcjUyEASYMKdVYK2DYAurKzQUqKDVEDowFFEfKFcy; Path=/; 
```

When I click on location I got the error:
`404 This page could not be found.`

Debugging:
GET `http://local.angeplus.com.br/iam-svc/public/self-service/errors?id=bf995dd5-6ec7-4cf0-bb2a-6c283d7e5742`
```
{
	"id": "bf995dd5-6ec7-4cf0-bb2a-6c283d7e5742",
	"error": {
		"code": 400,
		"debug": "key ory_kratos_oidc_auth_code_session does not exist in cookie: map[]\ngithub.com/ory/kratos/x.SessionGetString\n\t/home/ory/x/cookie.go:34\ngithub.com/ory/kratos/continuity.(*ManagerCookie).sid\n\t/home/ory/continuity/manager_cookie.go:99\ngithub.com/ory/kratos/continuity.(*ManagerCookie).container\n\t/home/ory/continuity/manager_cookie.go:109\ngithub.com/ory/kratos/continuity.(*ManagerCookie).Continue\n\t/home/ory/continuity/manager_cookie.go:64\ngithub.com/ory/kratos/selfservice/strategy/oidc.(*Strategy).validateCallback\n\t/home/ory/selfservice/strategy/oidc/strategy.go:215\ngithub.com/ory/kratos/selfservice/strategy/oidc.(*Strategy).handleCallback\n\t/home/ory/selfservice/strategy/oidc/strategy.go:259\ngithub.com/ory/kratos/selfservice/strategy.disabledWriter\n\t/home/ory/selfservice/strategy/handler.go:25\ngithub.com/ory/kratos/selfservice/strategy.IsDisabled.func1\n\t/home/ory/selfservice/strategy/handler.go:30\ngithub.com/ory/kratos/x.NoCacheHandler.func1\n\t/home/ory/x/nocache.go:18\ngithub.com/julienschmidt/httprouter.(*Router).ServeHTTP\n\t/go/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387\ngithub.com/ory/nosurf.(*CSRFHandler).handleSuccess\n\t/go/pkg/mod/github.com/ory/nosurf@v1.2.5/handler.go:201\ngithub.com/ory/nosurf.(*CSRFHandler).ServeHTTP\n\t/go/pkg/mod/github.com/ory/nosurf@v1.2.5/handler.go:158\ngithub.com/urfave/negroni.Wrap.func1\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:46\ngithub.com/urfave/negroni.HandlerFunc.ServeHTTP\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:29\ngithub.com/urfave/negroni.middleware.ServeHTTP\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38\ngithub.com/ory/kratos/x.glob..func1\n\t/home/ory/x/clean_url.go:12\ngithub.com/urfave/negroni.HandlerFunc.ServeHTTP\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:29\ngithub.com/urfave/negroni.middleware.ServeHTTP\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:38\ngithub.com/urfave/negroni.(*Negroni).ServeHTTP\n\t/go/pkg/mod/github.com/urfave/negroni@v1.0.0/negroni.go:96\ngithub.com/rs/cors.(*Cors).Handler.func1\n\t/go/pkg/mod/github.com/rs/cors@v1.6.0/cors.go:207\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2069\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2887\nnet/http.(*conn).serve\n\t/usr/local/go/src/net/http/server.go:1952\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1371",
		"reason": "No resumable session could be found in the HTTP Header.",
		"status": "Bad Request",
		"message": "session is not resumable"
	},
	"created_at": "2022-01-06T20:11:18.292048Z",
	"updated_at": "2022-01-06T20:11:18.292048Z"
}
```

Where's my mistake?

Save me!

**Reply by aeneasr** [reply]
From the looks of it, it looks correct. Is it possible that your cookie store has two conflicting cookies named `ory_kratos_continuity`?

**Reply by saulofilho** [reply]
I think not. 

When I did it from Insomnia, for example, was setted these two:
![cookies](https://user-images.githubusercontent.com/32524964/148448133-5f5bbfe5-b41b-4d83-b059-8dbf51964a3d.png)

After the last step, when I click on location, in anonymous (where the cookies are empty), only this one was setted:
![browser](https://user-images.githubusercontent.com/32524964/148448437-c695edb4-cfea-4295-a83e-639105e5f217.png)

Is it expected the `ory_kratos_continuity`, right? 

Thanks.

**Reply by aeneasr** [reply]
Why is the cookie set for the domain accounts.google.com. WTF? How can Kratos set a cookie for Google?

**Reply by aeneasr** [reply]
There's definitely something very, very funky going on.

**Reply by aeneasr** [reply]
Can you show the request which sets the cookie `ory_kratos_continuity` cookie? How does kratos escape the browser's security context and set a cookie for a completely different domain. wtf

**Reply by saulofilho** [reply]
Hahaha
Man, I'm struggle with that about 20 days and counting! 
I'll record the process. Give me a minute.

**Reply by aeneasr** [reply]
Is this chrome? If yes which version? Do you have anything else that's funky in your set up, like reverse proxies or something like that? Can you please share your full config also (omit secrets of course)?

**Reply by aeneasr** [reply]
Also what kratos version is this?

**Reply by aeneasr** [reply]
A colleague of mine said this looks like a non-standard browser / insomnia client. Can you try this flow in a regular browser? Does it give the same result?

**Reply by saulofilho** [reply]
Could you see it?

https://www.loom.com/share/56c9d412113c4e7387c8765992fd4b01

**Reply by saulofilho** [reply]
I was trying in Chrome Versão 96.0.4664.110 (Versão oficial) 64 bits.

Also in Firefox 95.0b6 (64-bit).

Kratos is 0.7.6-alpha.1.

Insomnia: 2021.7.2

Postman v9.8.0

**Reply by saulofilho** [reply]
No reverse proxies or something like that. 

My friends of work tried this as well. Nobody was successful.

**Reply by aeneasr** [reply]
Thank you for the screencast! I fear that clients such as insomnia get cookies often incorrect so we would need to reproduce this in the browser.

Is it possible for me to try this myself in chrome? The domain seems not to work for me (I guess it's running locally on your system?).

**Reply by aeneasr** [reply]
In the screen cast, you are doing half the flow in insomnia, the other half in chrome. This can't work - you need to execute the flow in the same tab, in the same browser, from start to finish!

**Reply by saulofilho** [reply]
Ops! Just an update: we are using reverse proxies with Ory/Oauthkeeper, said my coworker.

**Reply by saulofilho** [reply]
**.kratos-src.yaml**

```
version: v0.7.6-alpha.1

dsn: memory

serve:
  public:
    base_url: ${URL_IAM_SVC_PUBLIC}
    port: ${PORT_PUBLIC}
    cors:
      enabled: true
      allowed_origins:
        - ${URL_GATEWAY_SVC_PROXY}
      allowed_methods:
        - GET
      allowed_headers:
        - Content-Type
        - Accept
        - Cookie
      exposed_headers:
        - Content-Type
        - Set-Cookie
  admin:
    base_url: ${URL_IAM_SVC_ADMIN}
    port: ${PORT_ADMIN}

selfservice:
  default_browser_return_url: ${URL_GATEWAY_SVC_PROXY}/
  whitelisted_return_urls:
    - ${URL_GATEWAY_SVC_PROXY}

  methods:
    password:
      enabled: true
      config:
        ignore_network_errors: true
        haveibeenpwned_enabled: false
        max_breaches: 100
    oidc:
      enabled: true
      config:
        providers:
          - id: google
            provider: google
            client_id: ********************
            client_secret: ********************
            mapper_url: file:///home/ory/oidc.google.jsonnet
            scope:
              - email
              - profile
            requested_claims:
              id_token:
                email:
                  essential: true
                email_verified:
                  essential: true
          - id: facebook
            provider: facebook
            client_id: ********************
            client_secret: ********************
            mapper_url: file:///home/ory/oidc.facebook.jsonnet
            scope:
              - email
              - profile
            requested_claims:
              id_token:
                email:
                  essential: true
                email_verified:
                  essential: true

  flows:
    error:
      ui_url: ${URL_AUTH_WEB}/error

    settings:
      ui_url: ${URL_AUTH_WEB}/profile
      privileged_session_max_age: 15m

    recovery:
      enabled: true
      ui_url: ${URL_AUTH_WEB}/recovery

    verification:
      enabled: false
      ui_url: ${URL_AUTH_WEB}/verify
      after:
        default_browser_return_url: ${URL_GATEWAY_SVC_PROXY}/

    logout:
      after:
        default_browser_return_url: ${URL_AUTH_WEB}/login

    login:
      ui_url: ${URL_AUTH_WEB}/login
      lifespan: 10m

    registration:
      lifespan: 10m
      ui_url: ${URL_AUTH_WEB}/register
      after:
        password:
          hooks:
            - hook: session
        oidc:
          hooks:
            - hook: session

log:
  level: debug
  format: text
  leak_sensitive_values: true

secrets:
  cookie:
    - ********************
  default:
    - ********************

hashers:
  argon2:
    parallelism: 1
    memory: 128MB
    iterations: 2
    salt_length: 16
    key_length: 16

```

**Reply by aeneasr** [reply]
The problem is that you're executing half the flow in one system (Postman/Insomnia), and the other half in the browser. This can not work because the software does not share any cookies with one another. That's why the flow does not work. You need to run the flow from start (literally rendering the login form) to finish in one client.

**Reply by saulofilho** [reply]
> In the screen cast, you are doing half the flow in insomnia, the other half in chrome. This can't work - you need to execute the flow in the same tab, in the same browser, from start to finish!

Oh, nice. I will try that. And yes, it is local. 

Just an addendum: we coded the flow in Ruby and the tests responds with the same error.

**Reply by saulofilho** [reply]
> The problem is that you're executing half the flow in one system (Postman/Insomnia), and the other half in the browser. This can not work because the software does not share any cookies with one another. That's why the flow does not work. You need to run the flow from start (literally rendering the login form) to finish in one client.

Ok. I will do that. 

Thanks a lot for your time, man.

I gonna set this. I hope briefly bring some good news.

**Reply by saulofilho** [reply]
Here we go again!

Now I'm executing the flow in the same client. 
https://www.loom.com/share/f8ed6627f68d45cca1fb98ef65e05b6c

I've disabled the Follow Redirects to receive the Google screen.

**Reply by aeneasr** [reply]
The error is from Google, not Ory. The problem is that Insomnia might have some issues with Cookies. You need to try the flow in a proper browser, it will probably not work end-to-end in Postman, Insomnia, or these other HTTP tools because they are usually very bad at managing cookies.

**Reply by seunggs** [reply]
@aeneasr I'm getting this exact issue with Github now and I'm running everything in my browser.

@saulofilho Did you ever resolve this? 

This is what I'm doing in my local env (`http://localhost:3002` - Kratos is installed via helm in my EKS Kubernetes cluster and exposed via `emissary-ingress` mapping at `kratos.sidetrek.com`):

1) GET `https://kratos.sidetrek.com/self-service/registration/browser` -> this gives me back the flow info
2) POST `https://kratos.sidetrek.com/self-service/registration?flow=b28f7b8d-5161-48be-92a8-385ad1b24bae` -> this gives me `422 browser_location_change_required` error with `err.response.data.redirect_browser_to`, which I use to redirect
3) That's when I get the `400 session is not resumable` error with `No resumable session could be found in the HTTP Header.`.

I have the multiple subdomain setup via helm values:
```
config: {
  serve: {
    public: {
      base_url: `https://kratos.sidetrek.com`,
      domain_aliases: [
        {
          match_domain: `localhost:3002`,
          base_path: '/',
          scheme: 'http',
        },
        {
          match_domain: `sidetrek.com`,
          base_path: '/',
          scheme: 'https',
        },
        {
          match_domain: `staging.sidetrek.com`,
          base_path: '/',
          scheme: 'https',
        },
      ],
    }
  }
}
```

I also noticed that even though in the response from the POST call above I see the set-cookie response header with `ory_kratos_continuity` value set, I don't see anything saved in my cookies (if I look at it with Chrome browser's Cookies for `localhost:3002` domain). I am running Kratos in `development` mode.

What am I doing wrong?

**Reply by saulofilho** [reply]
Hi, @seunggs.

I didn't get. 
Therefore, my team had to take a step back to rebuild the engineering, because we have a flow which couldn't work with these aspects, for example an entity with multiple concerns, the microservices instances, the Oauthkeeper and other stuff.

**Reply by saulofilho** [reply]
We are currently working hard to produce a patch or a workaround to achieve this. When we got it, I'll post here.

**Reply by seunggs** [reply]
I see - thanks!

**Reply by aeneasr** [reply]
You need to make sure that everything is executed in the same browser, you can not do part on a server and another part on the browser. Also make sure that cookies are properly being set (in AJAX this is done with credentials-include) :)

**Comment by bruno-nascimento** [comment]
I had the same issue.
in `kratos.yml` (selfservice/methods/oidc/providers/github)i was using `127.0.0.1` and in github oauth application Authorization callback URL I was using `localhost`. I change it to 127.0.0.1 in github and it worked.

**Reply by vinckr** [reply]
Thanks for sharing your fix! Its important to keep the "localhost/127.0.0.1" consistent (in many cases such as OIDC)

**Comment by dissayamas1** [comment]
> Hi, guys.
> 
> I'm trying to implement a Social Login with Google but I'm receiving this error: `"code": 400,"status":"Bad Request","reason":"No resumable session could be found in the HTTP Header.","debug":"key ory_kratos_oidc_auth_code_session does not exist in cookie: map[]..."`
> 
> What `ory_kratos_oidc_auth_code_session` stands for?
> 
> Thank you.

{"data":{"created_at///2025-12-07T14:27:04.30243Z","error":{"code":400,"debug///key ory_kratos_oidc_auth_code_session ไม่มีคุกกี้: ory_kratos_continuity\ngithub.com/ory/kratos/x.SessionGetString.func1\n\t/go/src/github.com/ory/kratos/x/cookie.go:31\ngithub.com/ory/kratos/x.SessionGetString\n\t/go/src/github.com/ory/kratos/x/cookie.go:50\ngithub.com/ory/kratos/continuity.(*ManagerCookie).sessionID\n\t/go/src/github.com/ory/kratos/continuity/manager_cookie.go:119\ngithub.co m/ory/kratos/continuity.(*ManagerCookie).container\n\t/go/src/github.com/ory/kratos/continuity/manager_cookie.go:135\ngithub.com/ory/kratos/continuity.(*ManagerCookie).Continue\n\t/go/src/github.com/ory/kratos/continuity/manager_cookie.go:77\ngithub.com/ory/kratos/selfservice/strategy/oidc.(*Strategy).ValidateCallback\n\t/go/src/github.com/ory /kratos/selfservice/strategy/oidc/strategy.go:377\ngithub.com/ory/kratos/selfservice/strategy/oidc.(*Strategy).HandleCallback\n\t/go/src/github.com/ory/kratos/selfservice/strategy/oidc/strategy.go:458\ngithub.com/ory/kratos/selfservice/strategy.disabledWriter\n\t/go/src/github.com/ory/kratos/selfservice/strategy/handler.go:26\ngithub.com/ory/k ratos/selfservice/strategy/oidc.(*Strategy).setRoutes.IsDisabled.func1\n\t/go/src/github.com/ory/kratos/selfservice/strategy/handler.go:31\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/ory/kratos/x.(*RouterPublic).HandlerFunc.handleWithAllMiddlewares.func1.1\n\t/go/src/github.com/ory/kratos/x/router.go :195\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerResponseSize.func1\n\t/go/pkg/mod/github.com/prometheus/client_golang@v1.23.0/prometheus/promhttp/instrument_server.go:296\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/serv er.go:2322\ngithub.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerCounter.func1\n\t/go/pkg/mod/github.com/prometheus/client_golang@v1.23.0/prometheus/promhttp/instrument_server.go:147\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/prometheus/client_golang/prometheus/promhttp.Instrumen ระยะเวลาการจัดการ.func1\n\t/go/pkg/mod/github.com/prometheus/client_golang@v1.23.0/prometheus/promhttp/instrument_server.go:97\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerDuration.func2\n\t/go/pkg/mod/github.com/prometheus/client_golang@v1 .23.0/prometheus/promhttp/instrument_server.go:109\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/prometheus/client_golang/prometheus/promhttp.InstrumentHandlerRequestSize.func1\n\t/go/pkg/mod/github.com/prometheus/client_golang@v1.23.0/prometheus/promhttp/instrument_server.go:243\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2322\ngithub.com/ory/x/prometheusx.Metrics.Instrument.Metrics.instrumentHandlerStatusBucket.func1\n\t/go/src/github.com/ory/kratos/oryx/prometheusx/metrics.go:115\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2 322\ngithub.com/ory/x/prometheusx.(*ตัวบุคคลด้ามจับ).ServeHTTP\n\t/go/src/github.com/ory/kratos/oryx/prometheusx/middleware.go:42\ngithub.com/ory /kratos/x.(* สาธารณะ).HandlerFunc.handleWithAllMiddlewares.func1\n\t/go/src/github.com/ory/kratos/x/router.go:194\nnet/http.HandlerFunc.Ser veHTTP\n\t/usr/local/go/src/net/http/server.go:2322\nnet/http.(*ServeMux).ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2861\ngithub.com/ory/kratos/x.(*RouterPublic).ServeHTTP\n\t/go/src/github.com/ory/kratos/x/router.go:40\ngithub.com/ory/nosurf.(*CSRFHandler).handleSuccess\n\t/go /pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:234\ngithub.com/ory/nosurf.(*CSRFHandler).ServeHTTP\n\t/go/pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:185\ngithub.com/ory/kratos/cmd/daem on.servePublic.MaxBytesHandler.func4\n\t/usr/local/go/src/net/http/server.go:4100", "message": ไม่ได้รับการสนับสนุนที่กลับมาทำงานต่อได้ "" เหตุผล "" ไม่มีคุกกี้อีกครั้งเทียบกับการกลับมาทำงานต่อ ความปลอดภัยของความปลอดภัยและถูกบล็อก โปรดล้างคุกกี้และแคชของตัวคุณเองแล้วอีกครั้ง!"status"ลองร้องขอเป็นพิเศษ"},"id": "82ebf6c6-6c4c-4476-a11e-04a92598314b" "updated_at":2025-12-07T14:27:04.302 43Z"},"message": ดึงข้อมูลข่าวสารของโฟลว์สำเร็จแล้ว"} โปรดล้างคุกกี้และแคชของตัวเลือกของคุณแล้วลองอีกครั้ง! "สถานะ" ร้องขอต้องเป็น "}," id: "82ebf6c6-6c4c-4476-a 11e-04a92598314b"updated_at":2025-12-07T14:27:04.30243Z"},"message///ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}โปรดล้างคุกกี้และแคชของช่องของคุณแล้วลองอีกครั้ง !"สถานะ"ร้องขอเป็น"},"id":82ebf6c6-6c4c-4476-a11e-04a92598314b"updated_at""2025-12-07T14:27:04.30243Z"},"message: ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}com/ory/kratos/x/router.go:40\ngithub.com/ory/nosurf.(*CSRFHandler).handleSuccess\n\t/go /pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:234\ngithub.com/ory/nosurf.(*CSRFHandler).ServeHTTP\n\t/go/pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:185\ngithub.com/ory/kratos/cmd/daem on.servePublic.MaxBytesHandler.func4\n\t/usr/local/go/src/net/http/server.go:4100", "message": ไม่ได้รับการสนับสนุนที่กลับมาทำงานต่อได้ "" เหตุผล "" ไม่มีคุกกี้อีกครั้งเทียบกับการกลับมาทำงานต่อ ความปลอดภัยของความปลอดภัยและถูกบล็อก โปรดล้างคุกกี้และแคชของตัวคุณเองแล้วอีกครั้ง!"status"ลองร้องขอเป็นพิเศษ"},"id": "82ebf6c6-6c4c-4476-a11e-04a92598314b" "updated_at":2025-12-07T14:27:04.302 43Z"},"message": ดึงข้อมูลข่าวสารของโฟลว์สำเร็จแล้ว"} โปรดล้างคุกกี้และแคชของตัวเลือกของคุณแล้วลองอีกครั้ง! "สถานะ" ร้องขอต้องเป็น "}," id: "82ebf6c6-6c4c-4476-a 11e-04a92598314b"updated_at":2025-12-07T14:27:04.30243Z"},"message///ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}โปรดล้างคุกกี้และแคชของช่องของคุณแล้วลองอีกครั้ง !"สถานะ"ร้องขอเป็น"},"id":82ebf6c6-6c4c-4476-a11e-04a92598314b"updated_at""2025-12-07T14:27:04.30243Z"},"message: ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}com/ory/kratos/x/router.go:40\ngithub.com/ory/nosurf.(*CSRFHandler).handleSuccess\n\t/go /pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:234\ngithub.com/ory/nosurf.(*CSRFHandler).ServeHTTP\n\t/go/pkg/mod/github.com/ory/nosurf@v1.2.7/handler.go:185\ngithub.com/ory/kratos/cmd/daem on.servePublic.MaxBytesHandler.func4\n\t/usr/local/go/src/net/http/server.go:4100", "message": ไม่ได้รับการสนับสนุนที่กลับมาทำงานต่อได้ "" เหตุผล "" ไม่มีคุกกี้อีกครั้งเทียบกับการกลับมาทำงานต่อ ความปลอดภัยของความปลอดภัยและถูกบล็อก โปรดล้างคุกกี้และแคชของตัวคุณเองแล้วอีกครั้ง!"status"ลองร้องขอเป็นพิเศษ"},"id": "82ebf6c6-6c4c-4476-a11e-04a92598314b" "updated_at":2025-12-07T14:27:04.302 43Z"},"message": ดึงข้อมูลข่าวสารของโฟลว์สำเร็จแล้ว"} โปรดล้างคุกกี้และแคชของตัวเลือกของคุณแล้วลองอีกครั้ง! "สถานะ" ร้องขอต้องเป็น "}," id: "82ebf6c6-6c4c-4476-a 11e-04a92598314b"updated_at":2025-12-07T14:27:04.30243Z"},"message///ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}โปรดล้างคุกกี้และแคชของช่องของคุณแล้วลองอีกครั้ง !"สถานะ"ร้องขอเป็น"},"id":82ebf6c6-6c4c-4476-a11e-04a92598314b"updated_at""2025-12-07T14:27:04.30243Z"},"message: ดึงข้อมูลของโฟลว์สำเร็จแล้ว"}
